src/api/core/warren/tests/fixtures/db.py

```diff
@@ -20,7 +20,7 @@ def db_engine():
     SQLModel.metadata.create_all(engine)
 
     # Pretend to have all migrations applied
-    alembic_cfg = Config("core/warren/alembic.ini")
+    alembic_cfg = Config(settings.ROOT_PATH / "alembic.ini")
```

Comment:
Hence, we can use it as follow:

```suggestion
    alembic_cfg = settings.ALEMBIC_CFG